cypress/pages/home.page.ts

```diff
@@ -10,4 +14,35 @@ export const homePage = {
   openBankAccounts: () => {
     cy.get(homeLocator.bankAccountsButton).click();
   },
+  checkMoneyTransaction: (transactionType: String) => {
+    cy.fixture("transaction").then((transaction: any) => {
```

Comment:
It would be great to have typed  fixtures, since having `any` is a very brittle thing because you tumble upon "ccannot read proeprty 'something' of undefined"